Tests/ExampleAnalyzers/ExampleAnalyzer1/ExampleAnalyzer1.Test/ExampleAnalyzer1.Test.csproj

```diff
@@ -1,133 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <PropertyGroup>
```

Comment:
The two projects ExampleAnalyzer1.Test and ExampleAnalyzer1.Vsix are unused and have been removed.